website/src/tech-guide.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: layouts/page.njk
+title: CiviForm | Technical Readiness Guide
+---
+# CiviForm Technical Readiness Guide
+
+## Overview
+
+CiviForm is a powerful open-source tool for governments that helps simplify the administration of benefit programs and makes it easier for residents to access those benefits. It lets residents and community organizations find and apply to multiple services in one place.
+
+This doc is for a technical audience. Whoever will deploy CiviForm can use this  guide to learn how to deploy the tool on a cloud provider. It includes an overview and a detailed guide of all steps needed at each stage.
+
+To ensure a successful CiviForm implementation, read through the guide below and create your plan to deploy CiviForm.
+
+## Understanding Your Needs and Goals
+
+* Decide who will serve as the [CiviForm Admin](https://docs.civiform.us/overview/how-does-civiform-work#civiform-admin-experience), to lead efforts such as data integrations, across your civic entity.  
+  * Decide who will own and maintain your CiviForm deployment in the long term (an IT team, Digital Services, or other).  
+  * Make sure any engineers are added to all CiviForm engineering meetings.
+
+* Ensure that you have a “Technical Champion”, “Service Owner”, “Technical Project Manager”, or similar person to ensure proper integration into your IT infrastructure and program technical workflow (including understanding APIs, monitoring, and data integrations).
+
+* Security, Compliance, Data Privacy, and Access Controls  
+  * Security: Review the [CiviForm Security Brief](https://docs.google.com/document/d/1E2FeXpnpTF-JJiiszUhFDAgF8aPVTuE7HUbD245oTpg/edit) and ensure that you understand the necessary security reviews and processes that are required from your organization.  
+  * Data privacy: Implement robust security measures to protect citizen data.  
```

Comment:
not all applicants are citizens, right? I think we normally say "applicant" or "resident"

Reply:
Correct. This should say "applicants" (as I'm not sure about the residency requirement, if there even is one).

Reply:
Fixed in the original. It will fix the next time I do a PR for this doc. Thanks!